modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/search/bar/view.jsp

```diff
@@ -28,12 +29,33 @@ page import="com.liferay.portal.kernel.util.HtmlUtil" %><%@
 page import="com.liferay.portal.kernel.util.StringUtil" %><%@
 page import="com.liferay.portal.kernel.util.Validator" %><%@
 page import="com.liferay.portal.kernel.util.WebKeys" %><%@
-page import="com.liferay.portal.search.web.internal.search.bar.portlet.SearchBarPortletDisplayContext" %>
+page import="com.liferay.portal.search.web.internal.search.bar.portlet.SearchBarPortletDisplayContext" %><%@
+page import="com.liferay.portal.search.web.internal.search.bar.portlet.configuration.SearchBarPortletInstanceConfiguration" %>
+
+<%@ page import="java.util.ArrayList" %><%@
+page import="java.util.HashMap" %><%@
+page import="java.util.List" %><%@
+page import="java.util.Map" %>
 
 <liferay-theme:defineObjects />
 
+<portlet:defineObjects />
+
 <%
 SearchBarPortletDisplayContext searchBarPortletDisplayContext = (SearchBarPortletDisplayContext)java.util.Objects.requireNonNull(request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT));
+
+if (searchBarPortletDisplayContext.isDestinationUnreachable()) {
+	return;
+}
+
+SearchBarPortletInstanceConfiguration searchBarPortletInstanceConfiguration = searchBarPortletDisplayContext.getSearchBarPortletInstanceConfiguration();
+
+List<SearchBarPortletDisplayContext> entries = new ArrayList<>();
+
+Map<String, Object> contextObjects = new HashMap<String, Object>();
+
+contextObjects.put("namespace", renderResponse.getNamespace());
+contextObjects.put("searchBarPortletDisplayContext", searchBarPortletDisplayContext);
```

Comment:
What are your thoughts on moving this immediately before the `<liferay-ddm:template-renderer>` tag? Since these variables are only used for that taglib, it might more readable placing it next to where it's used.

Reply:
That's a good idea! I'll add it in.